include/boundary_standard.hxx

```diff
@@ -333,4 +333,38 @@ private:
   int width;
 };
 
+/// Convert toFieldAligned
```

Comment:
I've just added a bit more description, hopefully enough to make it clear what is happening.